lib/lan-connection.js

```diff
@@ -48,7 +47,7 @@ LAN.Connection = function(opts) {
   this.ssh = undefined;
 
   if (Tessel.isProvisioned()) {
-    this.auth.privateKey = opts.privateKey || fs.readFileSync(path.join(Tessel.LOCAL_AUTH_KEY));
+    this.auth.privateKey = opts.privateKey || fs.readFileSync(Tessel.LOCAL_AUTH_KEY);
```

Comment:
Should we write a common module that reads the auth key and throws the EACCESS error even in this case?. Then we can use it in both `lib/controller.js` and here.